dolphinscheduler-api-test/dolphinscheduler-api-test-case/src/test/java/org/apache/dolphinscheduler/api.test/pages/project/TaskInstancePage.java

```diff
@@ -0,0 +1,99 @@
+import org.apache.dolphinscheduler.api.test.core.Constants;
+import org.apache.dolphinscheduler.api.test.entity.HttpResponse;
+import org.apache.dolphinscheduler.api.test.entity.TaskInstanceResponseData;
+import org.apache.dolphinscheduler.api.test.entity.TaskInstanceResponseTotalList;
+import org.apache.dolphinscheduler.api.test.utils.JSONUtils;
+import org.apache.dolphinscheduler.api.test.utils.RequestClient;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+import java.util.HashMap;
+import java.util.Map;
+
+public final class TaskInstancePage {
+    private static final Logger logger = LoggerFactory.getLogger(TaskInstancePage.class);
+    private static String taskState = null;
+    private static Integer taskInstanceId = null;
+
+    public String queryTaskInstance(String sessionId, String projectName, String workFlowName){
+        Map<String, Object> params = new HashMap<>();
+        Map<String, String> headers = new HashMap<>();
+        params.put("pageSize", 10);
+        params.put("pageNo", 1);
+        params.put("searchVal", "");
+        params.put("processInstanceId", "");
+        params.put("host", "");
+        params.put("stateType", "");
+        params.put("startDate", "");
+        params.put("endDate", "");
+        params.put("executorName", "");
+        params.put("processInstanceName", workFlowName);
+
+        headers.put(Constants.SESSION_ID_KEY, sessionId);
+        RequestClient requestClient = new RequestClient();
+        ProjectPage project = new ProjectPage();
+        String projectCode = project.getProjectCode(sessionId, projectName);
+
+        WorkFlowDefinitionPage workflow = new WorkFlowDefinitionPage();
+        workflow.runWorkflow(sessionId, projectName, workFlowName);
+
+        HttpResponse res = requestClient.get("/projects/"+projectCode+"/task-instances", headers, params);
+
+
+        for (TaskInstanceResponseTotalList taskInstanceRes : JSONUtils.convertValue(res.body().data(), TaskInstanceResponseData.class).totalList()) {
+
+            taskState =  taskInstanceRes.state();
+            taskInstanceId = taskInstanceRes.id();
+        }
+
+        return taskState;
+
+    }
+
+    public HttpResponse queryTaskInstanceLog(String sessionId, String projectName, String workFlowName){
+        Map<String, Object> params = new HashMap<>();
+        Map<String, String> headers = new HashMap<>();
+
+        headers.put(Constants.SESSION_ID_KEY, sessionId);
+        RequestClient requestClient = new RequestClient();
+        TaskInstancePage taskInstance = new TaskInstancePage();
+        taskInstance.queryTaskInstance(sessionId, projectName, workFlowName);
+
+        params.put("taskInstanceId", taskInstanceId);
+        params.put("limit", 1000);
+        params.put("skipLineNum", 0);
+
+        HttpResponse res = requestClient.get("/log/detail", headers, params);
+        String res_log = (String) res.body().data();
+
+        logger.info("查询实例log：%s", res_log);
```

Comment:
Please remove unnessnary log.

Reply:
Thanks for the reminder, it has been fixed